### ISSUE TITLE
model accuracy

### ISSUE THREAD

**Issue by luanwd** [MITESHPUTHRANNEU/Speech-Emotion-Analyzer#43]
first, thanks for your great work. that let me quickly learn much domain knowledge about how to process voice data.but i am still troubled by the model result, i use your pre-trained model to test some vocie, i can get better performance.i want to reproduce your work, use the same data and same code as you do, but i can not get the same results, when i run about 100 epochs, the train loss still deline,but the test loss become to rise. i have tried many times and still no difference.so i want to ask what factors may result in this phenomenon?
